static/src/javascripts/projects/common/modules/commercial/creatives/gu-style-comcontent.js

```diff
@@ -25,12 +25,25 @@ define([
     var GustyleComcontent = function ($adSlot, params) {
         this.$adSlot = $adSlot;
         this.params  = params;
+        this.contentPosition = {
+            bottom: 'bottom: 30px;',
+            top: 'top: 30px;'
+        };
+        this.fontSize = {
```

Comment:
Why did you chose to go the inline styles way?

I'd prefer a set of classes:

Color:
default is black and for example `.gu-display-color--light` can be white color

Logo position:
`.gu-display-position--bottom-right`

You can see the pattern. To me it seems cleaner and more less coupled CSS with JS.